### DISCUSSION TITLE
uPad / MicroPad Text Editor

### DISCUSSION THREAD

**Discussion by djweinstein** [MicroPad/Web#367] [Q&A] [answered]
Hi. Just a question about the text editor in uPad. As I type in the editor, to add text to a note, the text does not wrap. Is there a way I can get the text to wrap while I type?

**Comment by NickGeek** [comment, marked as answer]
There will be a checkbox for it when https://github.com/MicroPad/MicroPad-Core/issues/193 is done, which will be part of MicroPad v4. In the meantime, you can hit enter to create a newline in your text. Single newlines are ignored so you can put them wherever you want to wrap.

**Reply by djweinstein** [reply]
Thank you very much for this info. I appreciate it.

**Reply by NickGeek** [reply]
@djweinstein Version 4 is out now on the web. https://web.getmicropad.com

You may need to clear your cache/refresh a few times to get the update immediately (otherwise you'll get it eventually)
